proposals/NNNN-Vehicle-data-additions.md

```diff
@@ -0,0 +1,1341 @@
+# Vehicle Data Additions
+
+* Proposal: [SDL-NNNN](NNNN-Vehicle-data-additions.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | SDL SERVER | SHAID | PROXY]
+
+## Introduction
+
+This proposal is to add new vehicle data items to SDL.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional set of vehicle data item through SDL. This goes in line with commitment to enhance SDL with even more rich vehicle data content.
+
+## Proposed Solution 
+
+We need to add some new vehicle data items. Following are the details on all the changed proposed.
+
+
+#### New vehicle data items:
+1. `WindowStatus`
+2. `SeatOccupancy`
+3. `StabilityControlsStatus`
+4. `GearStatus`
+5. `ClimateData`
+6. `handsOffSteering`
+7. `hvBatteryLevel`
+8. Additions to `BodyInformation` struct
```

Comment:
Each new vehicle data item should have its own proposal.  Please break this proposal up into separate proposals to ensure each new vehicle data item can be reviewed in detail by SDLC Members, as it’s unreasonable to expect that each OEM can simultaneously review if all of the new vehicle data structures will work with their vehicle data integrations or future integrations.  This is a similar concept to adding new Remote Control modules as individual proposals, as discussed here: https://github.com/smartdevicelink/sdl_evolution/issues/643#issuecomment-447925723.